include/internal/Hacl_Spec.h

```diff
@@ -40,6 +40,10 @@ extern "C" {
 #define Spec_Cipher_Expansion_Hacl_CHACHA20 0
 #define Spec_Cipher_Expansion_Vale_AES128 1
 #define Spec_Cipher_Expansion_Vale_AES256 2
+#define Spec_Cipher_Expansion_AESNI_PCLMUL_AES128 3
+#define Spec_Cipher_Expansion_AESNI_PCLMUL_AES256 4
+#define Spec_Cipher_Expansion_CT64_AES128 5
+#define Spec_Cipher_Expansion_CT64_AES256 6
```

Comment:
I do not see the corresponding source code change here: https://github.com/hacl-star/hacl-star/compare/pnmadelaine_aes...mamonet:hacl-star:pnmadelaine_aes is this handwritten?

Reply:
Yes, I didn't know the F* source of this module resides in `providers/evercrypt` directory. However, I can tell reflecting the changes in hacl side is a little bit challenging, hopefully we can address this tomorrow.